docs/components/content/examples/SlideoverExampleComponent.vue

```diff
@@ -0,0 +1,26 @@
+<script lang="ts" setup>
+const props = withDefaults(defineProps<{count: number}>(), {
```

Comment:
Would you mind using runtime declaration in `defineProps` and `defineEmits` for consistency?